.gitmodules

```diff
@@ -12,7 +12,7 @@
     url = https://github.com/philsquared/Catch.git
 [submodule "dynarmic"]
     path = externals/dynarmic
-    url = https://github.com/MerryMage/dynarmic.git
+    url = https://github.com/scowalt/dynarmic.git
```

Comment:
You can revert this now that the branch has been updated

Reply:
Done